cdap-app-templates/cdap-dq/src/main/java/co/cask/cdap/dq/DataQualityApp.java

```diff
@@ -137,6 +96,26 @@ public ConfigClass(int workflowScheduleMinutes, DataQualitySource source,
   @Override
   public void configure() {
     ConfigClass configObj = getContext().getConfig();
+    Preconditions.checkArgument(configObj.workflowScheduleMinutes > 0,
```

Comment:
can we open a jira to support more frequencies.  I think we would want to support a syntax like the one we use in stream source, '5m', '6h', '1d', etc.